client/src/components/Page.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { t } from "@lingui/macro";
+import { withI18n } from "@lingui/react";
+import { i18n } from "@lingui/core";
+import helpers from "../util/helpers";
+
+const metadataWithoutI18n = {
+  keywords: i18n._(t`pay rent, rent, can't pay rent, may rent, may 1`),
+  description: i18n._(
+    t`Use this free tool from JustFix.nyc to research your building and investigate landlords! We use property ownership mapping to identify a landlord's portfolio and provide data to indicate potential tenant harassment and displacement.`
+  ),
+  siteName: i18n._(t`Who owns what in nyc?`),
+};
```

Comment:
Hmm, so I was about to say that this top-level use if `i18n._()` isn't good, but then I realized that unlike the Tenant Platform, WoW doesn't do server-side rendering, so as long as we assume that the user's page will need to reload when they go from one locale to another (e.g. when they toggle languages it results in a full page reload, not just a react router history push) you should be ok.

That said, if we _do_ want to move to a SSR solution similar to the one that tenants2 has, this kind of code will make that harder to do, so it's up to you whether you want to wrap this kind of code in a function or not!

Reply:
Oh shoot. I thought that since I was actually rendering the `metadata` component as a function and not a constant, we'd be ok here but it sounds like we need to also make `metadataWithoutI18n` into a function as well?